### PR TITLE
New version: ojdkbuild.ojdkbuild version java-17-openjdk-17.0.1.0.12-1

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
\n\nAuto-updated by [vedantmgoyal2009/winget-pkgs-automation](https://github.com/vedantmgoyal2009/winget-pkgs-automation)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34317)

### CODE REVIEW

File: manifests/o/ojdkbuild/ojdkbuild/java-17-openjdk-17.0.1.0.12-1/ojdkbuild.ojdkbuild.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.7-1-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: ojdkbuild.ojdkbuild
+PackageVersion: java-17-openjdk-17.0.1.0.12-1
```
Comment (OfficialEsco) [inline manifests/o/ojdkbuild/ojdkbuild/java-17-openjdk-17.0.1.0.12-1/ojdkbuild.ojdkbuild.installer.yaml:5]:
```suggestion
PackageVersion: 17.0010.12.1
```

File: manifests/o/ojdkbuild/ojdkbuild/java-17-openjdk-17.0.1.0.12-1/ojdkbuild.ojdkbuild.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.7-1-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: ojdkbuild.ojdkbuild
+PackageVersion: java-17-openjdk-17.0.1.0.12-1
```
Comment (OfficialEsco) [inline manifests/o/ojdkbuild/ojdkbuild/java-17-openjdk-17.0.1.0.12-1/ojdkbuild.ojdkbuild.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 17.0010.12.1
```

File: manifests/o/ojdkbuild/ojdkbuild/java-17-openjdk-17.0.1.0.12-1/ojdkbuild.ojdkbuild.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.7-1-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: ojdkbuild.ojdkbuild
+PackageVersion: java-17-openjdk-17.0.1.0.12-1
```
Comment (OfficialEsco) [inline manifests/o/ojdkbuild/ojdkbuild/java-17-openjdk-17.0.1.0.12-1/ojdkbuild.ojdkbuild.yaml:5]:
```suggestion
PackageVersion: 17.0010.12.1
```

File: manifests/o/ojdkbuild/ojdkbuild/java-17-openjdk-17.0.1.0.12-1/ojdkbuild.ojdkbuild.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+PackageLocale: en-US
+Publisher: ojdkbuild open-source project
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: OpenJDK
```
Comment (OfficialEsco) [inline manifests/o/ojdkbuild/ojdkbuild/java-17-openjdk-17.0.1.0.12-1/ojdkbuild.ojdkbuild.locale.en-US.yaml:12]:
```suggestion
PackageName: ojdkbuild OpenJDK JRE
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/o/ojdkbuild/ojdkbuild/java-17-openjdk-17.0.1.0.12-1/ojdkbuild.ojdkbuild.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /passive
```
Comment (OfficialEsco) [inline manifests/o/ojdkbuild/ojdkbuild/java-17-openjdk-17.0.1.0.12-1/ojdkbuild.ojdkbuild.installer.yaml:10]:
```suggestion
  SilentWithProgress: /passive
ProductCode: {4BA86405-94A3-430D-93F2-82F6F78686AD}
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Comment by OfficialEsco** [comment]
I think we need to cleanup this package and split the versions

Also the issue:
` Manifest Error: Schema validation failed. Error context: <root>[ProductCode] Description: Value type not permitted by 'type' constraint. Error context: <root> Description: Failed to validate against schema associated with property name 'ProductCode'.  File: ojdkbuild.ojdkbuild.installer.yaml Manifest Error: Failed to process field. Field: ProductCode`

File: manifests/o/ojdkbuild/ojdkbuild/17.0010.12.1/ojdkbuild.ojdkbuild.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: ojdkbuild.ojdkbuild
+PackageVersion: 17.0010.12.1
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /passive
+ProductCode: {4BA86405-94A3-430D-93F2-82F6F78686AD}
```
Comment (ItzLevvie) [inline manifests/o/ojdkbuild/ojdkbuild/17.0010.12.1/ojdkbuild.ojdkbuild.installer.yaml:11]:
```suggestion
ProductCode: "{4BA86405-94A3-430D-93F2-82F6F78686AD}"
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**